### PR TITLE
This removes redundant tests which didn't add any value

### LINKED ISSUE #525: Withdraw.1.t.sol: rename/dedupe stale 'FailsIfNotOwner' auth tests
### Summary

`testWithdrawFailsIfNotOwner` (`contracts/test/Withdraw.1.t.sol:882`) and `testConsolidateFailsIfNotOwner` (`:676`) are carried over verbatim from the TVS suite, which uses an `onlyOwner` auth model. `Withdraw.1.sol` has no owner concept — `withdraw` is gated to `OperatorsRegistryAddress` and `consolidate` to `RiverAddress`.

The assertions are **correct** (they prank an unauthorized address and `expectRevert(Unauthorized(address))`), but:
- The `NotOwner` / `nonOwner` naming is misleading.
- They duplicate the better-named native tests:
  - `testWithdrawOnlyCallableByOperatorsRegistry` (`:321`) + `testWithdrawRevertsIfCalledByRiver` (`:333`)
  - `testConsolidateOnlyCallableByRiver` (`:352`)

Note the native `testWithdrawRevertsIfCalledByRiver` is strictly stronger — it proves River specifically cannot call `withdraw`, which the generic `FailsIfNotOwner` test does not.

### Suggested change

Either:
- Delete both `testWithdrawFailsIfNotOwner` / `testConsolidateFailsIfNotOwner` as redundant, or
- Rename to `testWithdrawRevertsIfCalledByUnauthorized` / `testConsolidateRevertsIfCalledByUnauthorized` and drop the `nonOwner` local naming.

### Severity

Trivial — test hygiene only, no behavioral gap.

### PR DESCRIPTION
**Pull request by mischat**
## Description

Fixes https://github.com/liquid-collective/liquid-collective-protocol/issues/525

This pull request removes two test cases from the `Withdraw.1.t.sol` test suite. Both tests previously verified that the `consolidate` and `withdraw` functions revert when called by an address other than the contract owner. The removal suggests that ownership checks may have changed or are no longer relevant for these functions.

Test removals related to access control:

* Removed the test `testConsolidateFailsIfNotOwner`, which checked that `consolidate` reverts when called by a non-owner address.
* Removed the test `testWithdrawFailsIfNotOwner`, which checked that `withdraw` reverts when called by a non-owner address.

## Notice

- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/liquid-collective/liquid-collective-protocol/pulls) for the same update/change?
- [ ] Have you assigned this PR to yourself?
- [ ] Have you added at least 1 reviewer?
- [ ] Have you updated the official documentation?
- [ ] Have you added sufficient documentation in your code?
- [ ] Have you added relevant tests to the official test suite?

## Pull Request Type

- [ ] 💫 New Feature (Breaking Change)
- [ ] 💫 New Feature (Non-breaking Change)
- [ ] 🛠️ Bug fix (Non-breaking Change: Fixes an issue)
- [ ] 🕹️ Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)

## Testing

- [ ] Have you tested this code with the official test suite?
- [ ] Have you tested this code manually?

## Summary by CodeRabbit

* **Tests**
  * Significantly expanded test suite for withdraw and consolidate operations with comprehensive coverage of input validation, fee calculation edge cases, refund safety mechanisms, error handling, and fuzzing-based invariant testing.

### CODE REVIEW

**Comment by mischat** [comment]
https://github.com/liquid-collective/liquid-collective-protocol/issues/525

File: contracts/test/Withdraw.1.t.sol
Comment (juliaaschmidt) [inline contracts/test/Withdraw.1.t.sol:1]: This one still has duplicates:

Maybe keep `testWithdrawOnlyCallableByOperatorsRegistry` only? 

```
    function testWithdrawOnlyCallableByOperatorsRegistry() external {
        bytes[] memory pubkeys = new bytes[](1);
        pubkeys[0] = VALID_PUBKEY_48;
        uint64[] memory amounts = new uint64[](1);
        amounts[0] = 1 gwei;
        address random = makeAddr("random");
        vm.deal(random, 10 gwei);
        vm.prank(random);
        vm.expectRevert(abi.encodeWithSignature("Unauthorized(address)", random));
        withdraw.withdraw{value: 1 gwei}(pubkeys, amounts, 1 gwei, excessFeeRecipient);
    }

    function testWithdrawRevertsIfCalledByRiver() external {
        bytes[] memory pubkeys = new bytes[](1);
        pubkeys[0] = VALID_PUBKEY_48;
        uint64[] memory amounts = new uint64[](1);
        amounts[0] = 1 gwei;
        vm.deal(address(river), 10 gwei);
        vm.prank(address(river));
        vm.expectRevert(abi.encodeWithSignature("Unauthorized(address)", address(river)));
        withdraw.withdraw{value: 1 gwei}(pubkeys, amounts, 1 gwei, excessFeeRecipient);
    }
```

File: contracts/test/Withdraw.1.t.sol [hunk 1]
```diff
@@ -858,23 +858,6 @@ contract WithdrawV1PectraTests is WithdrawV1TestBase {
         withdraw.consolidate{value: maxFeePerConsolidation * 2}(requests, maxFeePerConsolidation, excessFeeRecipient);
     }
 
-    /// @notice Tests that consolidate reverts when the caller is not River.
-    function testConsolidateFailsIfNotOwner() public {
```
Comment (juliaaschmidt) [inline contracts/test/Withdraw.1.t.sol:862]: Nice. 🔥  This one has all duplicates removed, the other remaining one is `testConsolidateOnlyCallableByRiver`

File: contracts/test/Withdraw.1.t.sol
Comment (mischat) [reply contracts/test/Withdraw.1.t.sol:1]: @juliaaschmidt i have removed it ... please close if happy

**Comment by mischat** [comment]
if you can approve this @juliaaschmidt feel free to merge it after !

**Review by juliaaschmidt — APPROVED** [review]

**Comment by mischat** [comment]
Yay
